### PR TITLE
[Snappi] Validate PFC frame bug fix

### PR DESCRIPTION
**Pull request by developfast**
### Description of PR

Summary: The validate PFC frame PR introduced a bug with a `TypeError` since the default parameters of the `config_capture_pkt` were not set correctly. This PR also fixes a frame checker bug where the result was always set to False. Both bugs have been fixed in this PR.  There is also a duration limit of pause flow frames added compared to the continuous flow of pause frames previously - this will improve the reliability of the snappi capture module since the capture module is heavily unreliable when traffic flows continuously. 

Fixes # (issue)

### Type of change

- [X] Bug fix
- [ ] Testbed and Framework(new/improvement)
- [ ] Test case(new/improvement)

### Back port request
- [ ] 201911
- [X] 202012
- [X] 202205

#### What is the motivation for this PR?
bug fix

#### How did you verify/test it?
tested on lab device

### CODE REVIEW

**Review by bingwang-ms — DISMISSED** [review]

**Comment by bingwang-ms** [comment]
LGTM

File: tests/snappi_tests/pfc/files/helper.py [hunk 1]
```diff
@@ -129,6 +130,9 @@ def run_pfc_test(api,
         # PFC pause frame capture is not requested
         valid_pfc_frame_test = False
 
+    pause_flow_dur_sec = DATA_FLOW_DURATION_SEC + data_flow_delay_sec + SNAPPI_POLL_DELAY_SEC + \
+        PAUSE_FLOW_DUR_BASE_SEC if valid_pfc_frame_test else 'continuous'
+
```
Comment (bingwang-ms) [inline tests/snappi_tests/pfc/files/helper.py:135]: If `valid_pfc_frame_test` is `False`, then you are adding a string to some integers.

Comment (developfast) [reply tests/snappi_tests/pfc/files/helper.py:135]: no this is just setting the mode to be fixed when this duration is passed in

Comment (developfast) [reply tests/snappi_tests/pfc/files/helper.py:136]: changed

**Review by bingwang-ms — APPROVED** [review]

**Pull request merged**